### PR TITLE
Add standard files

### PR DESCRIPTION
**Pull request by zippolyte**
*Note: Please remember to review the Datadog [Contribution Guidelines](https://github.com/DataDog/dd-agent/blob/master/CONTRIBUTING.md)
if you have not yet done so.*

### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Testing Guidelines

An overview on [testing](https://github.com/DataDog/dd-agent/blob/master/tests/README.md)
is available in our contribution guidelines.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Rules are matched bottom-to-top, so one team can own subdirectories
+# and another team can own the rest of the directory.
+
+# All your base
+*                       @DataDog/integrations-tools-and-libraries
+
+# Documentation
+*.md                    @DataDog/baklava @DataDog/integrations-tools-and-libraries
+LICENSE                 @DataDog/baklava @DataDog/integrations-tools-and-libraries
+LICENSE-3rdparty.csv    @DataDog/baklava @DataDog/integrations-tools-and-libraries
```
Comment (gzussa) [inline .github/CODEOWNERS]: Remove this line

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Configuration for https://github.com/actions/stale
```
Comment (gzussa) [inline .github/workflows/stale.yml]: Let's remove this GA since it doesn't work yet.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# Changelog
```
Comment (gzussa) [inline CHANGELOG.md]: We don't have one already 😱. Do you think it's possible to manually create it?

File: .github/CODEOWNERS (hunk 1 above)
Comment (zippolyte) [reply .github/CODEOWNERS]: why ?

File: CHANGELOG.md (hunk 3 above)
Comment (zippolyte) [reply CHANGELOG.md]: We'll have to manually create it. Right now, changelogs are only in releases descriptions, so it won't be hard at least

File: .github/CODEOWNERS (hunk 1 above)
Comment (gzussa) [reply .github/CODEOWNERS]: We don't have this file

Comment (gzussa) [reply .github/CODEOWNERS]: Actually we do! Nevermind

File: README.md [hunk 4]
```diff
@@ -64,31 +63,34 @@ Re-deploy the deployments to automatically add the agent.
...
+If the issue is with the agent itself, check out the [Datadog Agent repository][1] for issues and PR.
 
 # Support
 
-Support for this project is available via standard [Datadog support channels](http://docs.datadoghq.com/help/).
+Support for this project is available via standard [Datadog support channels][6].
 
 # License
 
 Apache 2.0 License
```
Comment (gzussa) [inline README.md]: Let's add a link

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| `duplicate`                   | Issue Only               | Duplicate issue.
+| `invalid`                     | Issue Only               | Invalid issue.
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
+
```
Comment (gzussa) [inline CONTRIBUTING.md]: We are missing `stale/exempt`

File: RELEASING.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Releasing
+
+Release can only be performed by Datadog maintainers of this repository.
+
+## Schedule
+
+Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule of one minor release every 6 weeks.
```
Comment (gzussa) [inline RELEASING.md]: Let's change it to 2 months.

Comment (zippolyte) [inline RELEASING.md]:
```suggestion
Releases of this project follow the [Datadog Agent](https://github.com/DataDog/datadog-agent) release schedule (+1/2 weeks).
```

**Pull request merged**